pkg/migration/controllers/migration.go

```diff
@@ -1791,26 +1794,26 @@ func (m *MigrationController) applyResources(
 
 	ruleset := resourcecollector.GetDefaultRuleSet()
 
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		return fmt.Errorf("error getting cluster config: %v", err)
+	}
+
+	srcClnt, err := apiextensionsclient.NewForConfig(config)
+	if err != nil {
+		return err
+	}
+	destClnt, err := apiextensionsclient.NewForConfig(remoteClient.remoteAdminConfig)
+	if err != nil {
+		return err
+	}
+
```

Comment:
You don't need to call NewForConfig again here. You can just use remoteClient.adminClient as  destClnt.

Reply:
destClnt is `*apiextensionsclient.Clientset` whereas remoteClient.adminClient is `*kubernetes.Clientset`. Hence keeping it.